idea_town/experiments/views.py

```diff
@@ -2,7 +2,7 @@
 from rest_framework.decorators import detail_route
 from rest_framework.response import Response
 from .models import (Experiment, ExperimentDetail)
-from .serializers import (ExperimentSerializer, ExperimentDeepSerializer,
```

Comment:
If we're not using the shallow `ExperimentSerializer`, it should probably get removed from `serializers.py` and `ExperimentDeepSerializer` just renamed to `ExperimentSerializer`.